README.md

```diff
@@ -83,8 +83,7 @@ model=HuggingFaceH4/zephyr-7b-beta
 # share a volume with the Docker container to avoid downloading weights every run
 volume=$PWD/data
 
-docker run --gpus all --shm-size 1g -p 8080:80 -v $volume:/data \
-3.0.0   ghcr.io/huggingface/text-generation-inference:3.0.0 --model-id $model
+docker run --gpus all --shm-size 1g -p 8080:80 -v $volume:/data ghcr.io/huggingface/text-generation-inference:3.0.0 --model-id $model
```

Comment:
Thanks! Maybe it's nicer to still have the `\` + newline + indentation in here for readability?